components/rules-watcher/rules-watcher.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
```

Comment:
You say this is a *Bash* script, but invoke it with `/bin/sh` - thats confusing. It feel wrong because `/bin/sh` won't run bash-isms.